doc/guide/install.rst

```diff
@@ -25,9 +25,6 @@ Prerequisites
   All dependencies will be downloaded from the internet.
   Do **not** use a metered, mobile connection!
 * Install `Anaconda`_, following the `installation instructions <https://docs.anaconda.com/anaconda/install/>`_ for your OS.
-* Create a **workspace directory**.
-  To make sure that your user can manipulate it without special privileges, use a subdirectory of your user/home directory.
-  Do **not** use a directory that is synchronized by cloud storage systems like OneDrive, iCloud or Polybox!
 * **Linux** users need to make sure they have ``git`` and ``curl`` installed.
   Ubuntu and Debian users may use APT:
```

Comment:
This is not needed anymore with the conda install. Should we move it to the section for the advanced installation?

Reply:
Indeed, will fix this!

Reply:
@chahank there is no need to download `env_climada.yml` as a single file anymore. I think we can ditch ``curl`` then, right?

Reply:
Good point! Agreed!

Reply:
Done